dev_guide/projected_volumes.adoc

```diff
@@ -0,0 +1,170 @@
+//
+
+.Pod with a secret, a downward API, and a configmap
+====
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/my-username
+      - downwardAPI:
+          items:
+            - path: "labels"
+              fieldRef:
+                fieldPath: metadata.labels
+            - path: "cpu_limit"
+              resourceFieldRef:
+                containerName: container-test
+                resource: limits.cpu
+      - configMap:
+          name: myconfigmap
+          items:
+            - key: config
+              path: my-group/my-config
+----
+====
+
+.Pod with multiple secrets with a non-default permission mode set
+====
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/my-username
+      - secret:
+          name: mysecret2
+          items:
+            - key: password
+              path: my-group/my-password
+              mode: 511
+----
+
+====
+
+Each projected volume source is listed in the spec under `sources`. 
+
+[NOTE]
+====
+The `defaultMode` can only be specified at the projected level and not for each
+volume source. However, as illustrated above, you can explicitly set the `mode`
+for each individual projection.
+====
+
+## Configure a projected volume for a pod
+
+The following example shows how to use a projected volume to mount several existing volume sources into the same directory. 
+
+The steps can be used to create a username and password xref:../../dev_guide/secret.adoc#dev-guide-secret[Secrets] from local files. You then create a pod that runs one container, using a projected volume to mount the Secrets into the same shared directory.
+
+. Create a pod configuration file similar to the following:
++
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-projected-volume
+spec:
+  containers:
+  - name: test-projected-volume
+    image: busybox
+    args:
+    - sleep
+    - "86400"
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: user
+      - secret:
+          name: pass
+----
+
+. Create files containing the secrets:
++
+----
+echo -n "admin" > ./username.txt
+echo -n "1f2d1e2e67df" > ./password.txt
+----
```

Comment:
The user and pass names are used in https://github.com/kubernetes/kubernetes.github.io/pull/3856/files#diff-92b561a119469ea9519cb2bbe83437d9R21, which I think you left out. The values are made up and can be anything valid (base64 encoded).